src/components/common/header.js

```diff
@@ -57,15 +57,17 @@ const Header = (props) => {
       </h3>
       {adminMenu && (
         <div className="dropdown">
-          <p>Edit Users</p>
+          <p onClick={()=>{
```

Comment:
Dan pointed something out about this but I don't know why you can't or shouldn't do this